bodhi/server/util.py

```diff
@@ -1303,3 +1303,14 @@ def pyfile_to_module(
         e.strerror = 'Unable to load file (%s)' % e.strerror
         raise
     return d
+
+
+def json_escape(text: str) -> str:
+    """Escape double quotes for JSON.parse compatibility.
+
+    Args:
+        text (str): The text to sanitize.
```

Comment:
Since you are using type annotations, let's drop 'str' here.

Reply:
corrected, thanks